api/rest/v1/bulk_info_place.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Info on a place
+nav_exclude: true
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/info/place
+---
+
+
+
+## /v1/bulk/info/place
+
+Get basic information about multiple [places](/api/rest/v1/getting_started#place).
+
+This API returns basic information on multiple places, given each of their DCIDs. The information provided is per place, and includes the place's name, type (city, state, country, etc.), as well as information on all parent places that contain the place queried.
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Tip:</b><br />
+   For a rich, graphical exploration of places available in the Data Commons knowledge graph, take a look at the [Place Explorer](https://datacommons.org/place).
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    To get information on a variable instead of a place, see [/v1/bulk/info/variable](/api/rest/v1/info/variable).<br />
+    For querying a single variable and a simpler output, see the [simple version](/api/rest/v1/info/place) of this endpoint.
+</div>
+
+
+
+## Request
+
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">GET Request</button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">POST Request</button>
+</div> 
+
+<div id="GET-request" class="api-tabcontent api-signature"><div class="scroll">
```

Comment:
nit: have the inner <div> indented in newline

Reply:
fixed!